packages/devtools_app/lib/src/screens/network/har_data_entry.dart

```diff
@@ -133,7 +137,7 @@ class HarDataEntry {
       // Response
       NetworkEventKeys.response.name: <String, Object?>{
         NetworkEventKeys.status.name: e.status,
-        NetworkEventKeys.statusText.name: '',
+        NetworkEventKeys.statusText.name: e.general[NetworkEventKeys.reasonPhrase.name]??'',
```

Comment:
run the dart formatter from the `devtools_app` directory: `dart format .`

Reply:
this is done